edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -38,7 +39,7 @@ import FancyButton from '../input/FancyButton';
  * @param {boolean} isModal - boolean to check if the screen is a modal then change the style
  * @returns {ReactComponent} Screen to see a card
  */
-const CardScreenComponent: ReactComponent<{ deckId: string, cardIndex: number; isModal?: boolean; }> = ({ deckId, cardIndex, isModal }) => {
+const CardScreenComponent: ReactComponent<{ deckId: string, cardIndex: number; isModal?: boolean; modalRef: React.RefObject<BottomSheetModal> }> = ({ deckId, cardIndex, isModal, modalRef }) => {
```

Comment:
Could you explain why you use React.RefObject here?

Reply:
I use it because in CardScreenComponent there is an option to edit or delete the card from the modal view so it needs to close the modal when the user wants to edit and it needs to close the modal after the card is deleted. That's why I have to give CardScreenComponent the modal :D